src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -75,10 +75,17 @@
     <None Include="**\windows\**\*.cs" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
   </ItemGroup>
 
+  <ItemGroup Condition="$(TargetFramework.StartsWith('netstandard')) != true AND '$(TargetFramework)' != 'net6.0'">
```

Comment:
I think we can remove it because we do not have netstandard target framework

Reply:
Agreed. We already target `net6.0` which has become the new .NET Standard now that we have `net6.0-ios` and `net6.0-android`,

Reply:
See this [comment](https://github.com/CommunityToolkit/Maui/pull/290#discussion_r807389661)

When we implement the `PlatformConfiguration` features we will need to implement handlers on this project and this rule will be needed.